app/models.py

```diff
@@ -750,6 +759,112 @@ def serialize(self, include_user=False):
 
         return data
 
+
+class Brief(db.Model):
+    __tablename__ = 'briefs'
+
+    STATUSES = ('draft', 'live')
+
+    id = db.Column(db.Integer, primary_key=True)
+
+    framework_id = db.Column(db.Integer, db.ForeignKey('frameworks.id'), nullable=False)
+    _lot_id = db.Column("lot_id", db.Integer, db.ForeignKey('lots.id'), nullable=False)
+
+    data = db.Column(JSON)
+    status = db.Column(db.String, index=False, unique=False,
+                       nullable=False, default='draft')
+
+    created_at = db.Column(db.DateTime, index=True, nullable=False,
+                           default=datetime.utcnow)
+    updated_at = db.Column(db.DateTime, index=True, nullable=False,
+                           default=datetime.utcnow, onupdate=datetime.utcnow)
+
+    __table_args__ = (db.ForeignKeyConstraint([framework_id, _lot_id],
+                                              ['framework_lots.framework_id', 'framework_lots.lot_id']),
+                      {})
+
+    users = db.relationship('User', secondary='brief_users')
+    framework = db.relationship('Framework', lazy='joined')
+    lot = db.relationship('Lot', lazy='joined')
+
+    @validates('users')
+    def validates_users(self, key, user):
+        if user.role != 'buyer':
+            raise ValidationError("The brief user must be a buyer")
+        return user
+
+    @property
+    def lot_id(self):
+        return self._lot_id
+
+    @lot_id.setter
+    def lot_id(self, lot_id):
+        raise ValidationError("Cannot update lot_id directly, use lot relationship")
+
+    @validates('lot')
+    def validates_lot(self, key, lot):
+        if not lot.requires_brief:
+            raise ValidationError("Lot '{}' does not require a brief".format(lot.name))
+        return lot
+
+    @validates('data')
+    def validates_data(self, key, data):
+        data = drop_foreign_fields(data, [
+            'id',
+            'frameworkSlug', 'frameworkName', 'frameworkStatus',
+            'lot', 'lotName',
+            'updatedAt', 'createdAt', 'links'
+        ])
+
+        data = strip_whitespace_from_data(data)
+        data = purge_nulls_from_data(data)
+
+        return data
+
+    @validates('status')
+    def validates_status(self, key, status):
+        if status not in self.STATUSES:
+            raise ValidationError("Invalid brief status '{}'".format(status))
+        if self.status == 'live' and status == 'draft':
+            raise ValidationError("Cannot make a live brief draft")
+        return status
+
+    def update_from_json(self, data):
+        current_data = dict(self.data.items())
+        current_data.update(data)
+
+        self.data = current_data
+
+    def serialize(self):
+        data = dict(self.data.items())
+
+        data.update({
```

Comment:
The frontend will need to check that a user belongs to a brief and can view/modify/publish, so we need to return a list of brief users here (maybe only when an individual brief is requested).